CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 1.1.0 (2021-02-03)
+## Product-wide changes
+* **Remove support for Ubuntu 16.04, add support for 18.04**. Ubuntu will soon end their support for 16.04, so we're changing our support to match.
```

Comment:
Suggest saying this bumps the minimum support to 18.04 to match rather than "add support for 18.04".  If the "add" is in the context of the build/test pipelines then wouldn't that represent the min version?  It might also be the max at the moment too.

Reply:
Fixed